cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/detector/AnomalyDetectorState.java

```diff
@@ -107,53 +108,66 @@ private static String getDateFormat(long timeMs) {
     return formatter.format(date);
   }
 
-  private static void populateCommonDetails(AnomalyState anomalyState, Map<String, Object> anomalyDetails, boolean useDateFormat) {
-    anomalyDetails.put(useDateFormat ? DETECTION_DATE : DETECTION_MS,
-                       useDateFormat ? getDateFormat(anomalyState.detectionMs()) : anomalyState.detectionMs());
+  private static void populateCommonDetails(AnomalyState anomalyState, Map<String, Object> anomalyDetails, boolean isJson) {
+    anomalyDetails.put(isJson ? DETECTION_DATE : DETECTION_MS,
+                       isJson ? anomalyState.detectionMs() : getDateFormat(anomalyState.detectionMs()));
     anomalyDetails.put(STATUS, anomalyState.status());
     anomalyDetails.put(ANOMALY_ID, anomalyState.anomalyId());
-    anomalyDetails.put(useDateFormat ? STATUS_UPDATE_DATE : STATUS_UPDATE_MS,
-                       useDateFormat ? getDateFormat(anomalyState.statusUpdateMs()) : anomalyState.statusUpdateMs());
+    anomalyDetails.put(isJson ? STATUS_UPDATE_DATE : STATUS_UPDATE_MS,
+                       isJson ? anomalyState.statusUpdateMs() : getDateFormat(anomalyState.statusUpdateMs()));
```

Comment:
Is the mapping is reversed, should it be
`STATUS_UPDATE_DATE` -> `getDateFormat(anomalyState.detectionMs())`
`DETECTION_MS` -> `anomalyState.statusUpdateMs()`